src/main.c

```diff
@@ -89,99 +89,50 @@ static void write16_be(uint8_t *raw, uint16_t value)
 	raw[1] =  value & 0xff;
 }
 
-static void
```

Comment:
I think I've seen this pattern elsewhere in OpenWrt uci parsing code -- maybe it exists to standardize it and make it machine-readable? -- but --

1) I might be wrong and this part of `realtek-poe` might just be a blogicism
2) Even if I found this pattern in other uci / ubus parsing code, I don't think I'd be able to call up the right people to explain their patterns.

So this has my blessing.